pyscript.core/docs/README.md

```diff
@@ -314,9 +314,9 @@ Whenever computing relatively expensive stuff, such as a *matplot* image, or lit
 
 `@pyscript/core` adds a functionality called `XWorker` to all of the interpreters it offers, which works in each language the way `Worker` does in JavaScript.
 
-In each Interpreter, `XWorker` is a global reference, with a counter `xworker` (lower case) global reference within the worker code.
+In each Interpreter, `XWorker` is either global reference, or a `from xworker import XWorker` module's utility, with a counter `xworker` (lower case) global reference, or a `from xworker import xworker` module's utility, within the worker code.
```

Comment:
I think I misunderstood this during our call today. I thought we'd be removing `XWorker` as a global reference and only keeping it as part of the module namespace. If we are keeping both, what users should use and why?

Reply:
Not that it's a big deal (especially right now), it's just a way to clarify and make sure we are pointing users at the "right way of doing things"

Reply:
I read this as meaning that it depends on the runtime? So if the runtime doesn't have a convenient way to import xworker, it would just be added as a global?

Reply:
@WebReflection I think it would be good to say "or as an import" or something like that since the wording "a `from xworker import xworker` is Python specific.

Reply:
@fpliger what @hoodmane said ... it's **not possible** in *Wasmoon* or *Ruby* to register a *JS* module and consume it ... in those cases (code should speak for itself) stuff is added globally, as there's **no other way** to do so.

> I think it would be good to say "or as an import" or something like that since the wording "a from xworker import xworker is Python specific.

that's true but at the end of the day, we only have *Python* interpreters defined as stable, and these are the only one able to import modules ... so I've thought that having the wording 100% matching the code needed to make it work at this stage was convenient. I can change that though.

Reply:
I will try to look into adding `registerJSModule` or something like it to wasmoon if I can get around to it.

Reply:
Yeah. What threw me off was the wording above. It reads as both are possibilities with **any** interpreter not that it's available as import for interpreters that support it and if the runtime doesn't have a convenient way to import xworker, it would just be added as a global